src/platform/middleware.ts

```diff
@@ -73,3 +75,16 @@ export const gracefullyDenyDeletedDevices: RequestHandler = (
 			next();
 		});
 };
+
+export const registerDeviceContact = (path: string) => {
```

Comment:
Can you add a
```ts
path = _.toPath(path)
```
this will resolve the path once and avoid paying that cost every time the returned function is called (only an issue because the function will be called a lot) - the alternative would be to use `registerDeviceContact(['params', 'uuid']),` but it's nicer to avoid there even being the possibility of a performance pitfall